source/change-streams/change-streams.rst

```diff
@@ -354,6 +353,8 @@ Command syntax:
       ...
     }
 
+Drivers MUST use the ``ns`` returned in the ``aggregate`` command to set the ``collection`` option in subsequent ``getMore`` commands.
```

Comment:
Under what circumstances would this not be the same as the collection used to send the original aggregate command? Is this some special database/client watch implementation thing?

Reply:
OK.  Didn't realize but I agree we ought to pass back what we get.

Reply:
This is similar to what drivers must do for listCollections.